src/simulation/power/simpleBattery/simpleBattery.h

```diff
@@ -39,6 +39,8 @@ class SimpleBattery: public PowerStorageBase {
 
 public:
     double storageCapacity; //!< [W-s] Battery capacity in Watt-seconds (Joules).
+    bool fault = false; //!< Fault flag
+    double faultCapacityRatio = 1.0; //!< Fault capacity ratio
```

Comment:
explain what 1.0 default value means here.  This means by default we have full capacity, correct?